garden-service/test/unit/src/commands/get/get-task-result.ts

```
@@ -0,0 +1,44 @@
+import { dataDir, makeTestGarden, expectError } from "../../../../helpers"
+import { resolve } from "path"
+import { GetTaskResultCommand } from "../../../../../src/commands/get/get-task-result"
+import { expect } from "chai"
+
+describe("GetTaskResultCommand", () => {
+  it("should not find module by module name", async () => {
```

The message doesn't really make sense. Shouldn't it be something like: "should throw error if task not found"